.github/workflows/android.yml

```diff
@@ -0,0 +1,23 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ main, dev ]
+  pull_request:
+    branches: [ main, dev ]
```

Comment:
We can use `branches: '*'` so that the CI action runs when we have a hierarchy of PRs

Reply:
Would that mean it will run the build for every branch?

Reply:
Sorry I wasn't clear above, I meant this

```
  pull_request:
    branches: '*'
```

This will run the workflow on all Pull Requests.

Currently CI will run only for the PRs that are made against main/dev.

Reply:
No no, that I understood. I was wondering if it is necessary to run the CI for every branch on a PR or we should just run it on major branches like main and dev. Anyways, I am doing it for all the branches. Let me know if any other change is required or are we good to merge?